### DISCUSSION TITLE
Add an option for auto equalizing splits

### DISCUSSION THREAD

**Discussion by miranquil** [ghostty-org/ghostty#4052] [Feature Requests, Ideas] [answered]
Say, when I press "Cmd+D", ghostty creates splits on the right side for now.

Thinking that an auto-equalizing option would be great, so it will be no need to call something like "split then equalize".

**Comment by JakeWharton** [comment, marked as answer]
Dupe of https://github.com/ghostty-org/ghostty/discussions/3839
